_posts/camazotz/2016-01-18-NavaneetsTurtleProgram

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+author: camazotz
+title: "Navaneet's Turtle Programs!"
+---
+
+Turtle Drawing Exercise:
+
+<iframe src="https://trinket.io/embed/python/7fa0f33bdb" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Turtle Hacking Exercise:
+
+<iframe src="https://trinket.io/embed/python/17cdca3460" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Reflection:
+
+I liked both of these exercises quite a bit. The first exercise helped me get familiar with the Turtle API and the second one allowed me to experiment with different functions. I was originally trying to simulate a "race" between four turtles where the background would change colors (from 'red' to 'orange' to 'green') to indicate 'Ready, set, go!'. I had each turtle at a different speed but I was having trouble figuring out how to make more than one turtle move at the same time so I scrapped that idea and went for drawing a smiley face with three turtles. I kept the background color change since it added variety and changed the last color to Carolina blue instead of green. I looked through the [Turtle API] (https://docs.python.org/3.1/library/turtle.html#turtle.hideturtle) to learn how to do this.
```

Comment:
Nice work!  Synchronous motion of turtles requires some more advanced stuff we won't get to for a while.  Look at the source of this example program if you want to try to teach yourself some: https://trinket.io/python/88dd6c94d1